.github/workflows/wheel_linux_aarch64.yml

```
@@ -161,7 +162,7 @@ jobs:
           regex: '.*[0-9]+.[0-9]+.[0-9]+[-_]?rc[0-9]+'
 
       - uses: actions/upload-artifact@v3
-        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
+        # if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```

```suggestion
        if: ${{ github.event_name == 'release' || github.ref == 'refs/heads/master' }}
```
Uncomment before merging.